city_metrix/layers/albedo.py

```diff
@@ -118,7 +121,15 @@ def calc_s2_albedo(image):
         ## S2 MOSAIC AND ALBEDO
         dataset = get_masked_s2_collection(ee.Geometry.BBox(*bbox), self.start_date, self.end_date)
         s2_albedo = dataset.map(calc_s2_albedo)
-        albedo_mean = s2_albedo.reduce(ee.Reducer.mean())
+        if self.resampling_method is not None:
+            albedo_mean = (s2_albedo
+                           .map(lambda x: set_resampling_method(x, self.resampling_method))
```

Comment:
small nitpick to reduce code repetition, but I think because of how GEE works, you could just do this more like 

```
resampled_albedo = s2_albedo
if self.resampling_method is not None:
            resampled_albedo = s2_albedo
                           .map(lambda x: set_resampling_method(x, self.resampling_method))

albedo_mean = (resampled_albedo.
                           .reduce(ee.Reducer.mean())
                           )
```

Maybe that's not any better though... your choice

Reply:
Alternatively, based on comment below, maybe you could just define the resampling in the Xee call and just pass it along to `get_image_collection`?

Reply:
As far as I have determined, XEE does not yet directly support resampling. Resampling must instead be specified on the image.

Reply:
I found a way to fully implement in the called function.